docker-compose.yml

```diff
@@ -3,7 +3,7 @@
 services:
 
   redis-stanalone:
-    image: redislabs/client-libs-test:8.0-M02
+    image: redislabs/client-libs-test:8.0-M03
```

Comment:
Given the fact that 8.0-M03 brings more things to implement (ACL + CONFIG), I think it makes sense to make it as a separate PR, WDYT?

Reply:
I agree.